src/pudl/metadata/resources/eia923.py

```diff
@@ -20,6 +21,7 @@
                 "plant_id_eia",
                 "boiler_id",
                 "energy_source_code",
+                "prime_mover_code",
```

Comment:
Just for readability in the default outputs, do we prefer for this to show "all the prime movers within a fuel type" which is what this will show, or do we prefer "all the fuels consumed by a given type of prime mover" which would mean swapping the order of `energy_source_code` and `prime_mover_code`?

Reply:
This is a good question. But I don't think there is a right or wrong answer here as the helpfulness of the nesting depends on the plant. If a plant has one ESC but two PMs, the current order will be more visually helpful and vice versa. On quick investigation, there are substantially more plants in this table with one ESC, so I think this ordering makes more sense.